src/middlewares/dedupleMiddleware.js

```diff
@@ -0,0 +1,31 @@
+import objectHash from 'object-hash';
+
+/**
+ * @return {Middleware}
```

Comment:
hum it doesn't really returns a middlewares, this this the middlewares but when you have middlewares options you need to create a factory that returns a middleware